### ISSUE TITLE
Replace images with CSS3?

### ISSUE THREAD

**Issue by dandv** [wayfarer/WowWindow#1]
I was wondering if it would possible to not need any image dependencies, and instead use CSS-based icons?

There's an awesome CSS icon set at http://nicolasgallagher.com/pure-css-gui-icons/demo/

**Comment by wayfarer** [comment]
Yeah, it's possible, but is it really needed?  Also, what are the compatibility issues with IE?  It absolutely needs to support the last 3 generations of IE.  I'd label this a future scope issue, as it will be more practical in the future.
